samples/LocalizationSample/Startup.cs

```diff
@@ -23,31 +22,19 @@ public void ConfigureServices(IServiceCollection services)
 
         public void Configure(IApplicationBuilder app, IStringLocalizer<Startup> SR)
         {
-            var supportedCultures = new List<CultureInfo>
-            {
-                new CultureInfo("en-US"),
-                new CultureInfo("en-AU"),
-                new CultureInfo("en-GB"),
-                new CultureInfo("es-ES"),
-                new CultureInfo("ja-JP"),
-                new CultureInfo("fr-FR"),
-                new CultureInfo("zh"),
-                new CultureInfo("zh-CN")
-            };
-            var options = new RequestLocalizationOptions
-            {
-                DefaultRequestCulture = new RequestCulture("en-US"),
-                SupportedCultures = supportedCultures,
-                SupportedUICultures = supportedCultures
-            };
-            // Optionally create an app-specific provider with just a delegate, e.g. look up user preference from DB.
-            // Inserting it as position 0 ensures it has priority over any of the default providers.
-            //options.RequestCultureProviders.Insert(0, new CustomRequestCultureProvider(async context =>
-            //{
-
-            //}));
+            var supportedCultures = new [] { "en-US", "en-AU", "en-GB", "es-ES", "ja-JP", "fr-FR", "zh", "zh-CN" };
+            app.UseRequestLocalization(options =>
+                options
```

Comment:
I realize that this is based on my own suggestion https://github.com/aspnet/Localization/issues/127 - (from 2.5 years ago wow!), but I'm going to suggest a modification on what I said.

One comment here for posterity - the design of how work with these kinds of builders has changed since I made the proposal back in the much simpler time of 2015.

---
The typical pattern is:

```
services.AddLocalization(...) // returns a builder - that builder will modify options
```

```
app.UseRequestLocalization(); // no args version gets options from services
```
---

Unfortunately, we already have a `services.AddLocalization()` and it returns `IServiceCollection`. Changing this would be a breaking change and we would not do that for minor API polish like this. So we've painted ourselves into a corner. 

I think this PR based on my original proposal is the best we can do. But I wanted to point out that it's not consistent with our other API shapes.